<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SampleConsole</class>
 <widget class="QMainWindow" name="SampleConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sample Console</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>801</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="hostEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="portSpinBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>63</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendNameButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendDataButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendPatchButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Patch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="patchButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Patch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unpatchButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Unpatch</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>781</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="idSpinBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>230</y>
      <width>63</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="completeCheckBox">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>230</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Complete</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>240</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectPushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>270</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deselectPushButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>270</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Deselect</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
